### ISSUE TITLE
patch release 1.0.1

### ISSUE THREAD

**Issue by orbeckst** [alchemistry/alchemlyb#279]
We have a number of bug fixes that should be released soon. 

Let's get 1.0.1 out before doing anything else.

**Comment by xiki-tempula** [comment]
I'm on holiday this week to use up my holidays before the year ends so have some time to do some work.
So my plan is to first fix Fix #206 through #278
Then another PR to refactor the tests for the ABFE as it is kind of organised in a very chaotic way.
Then I will fix #276 when the tests for the ABFE is more organised so the new test could be added in a more clear fashion.
Then I think we could release 1.0.1

**Comment by xiki-tempula** [comment]
I think the 1.0.1 is ready to go now.

**Comment by xiki-tempula** [comment]
Sorry, might need to wait to see if https://github.com/alchemistry/flamel/issues/24 needs fixing.

**Comment by orbeckst** [comment]
I'll do a 1.0.1, it's easy enough. We can do a 1.0.2 if the flamel issue is due to anything in alchemlyb.

**Comment by xiki-tempula** [comment]
Sorry for the last minute change, I'm working on https://github.com/alchemistry/alchemlyb/pull/268, do we want to squeeze in a DeprecationWarning for autombar?

**Comment by orbeckst** [comment]
Yes, I only just saw #284. I haven't done the release yet so I'll wait for PR #285 . (I'll change the tag etc once everything is ready.)

**Comment by orbeckst** [comment]
Might as well do minor doc fixes PR #286

**Comment by orbeckst** [comment]
1.0.1 is on pypi and thanks to the automerged PR https://github.com/conda-forge/alchemlyb-feedstock/pull/7 also conda-forge.
